### PR TITLE
Updates for AudioVideo plugin

### PR DESCRIPTION
**Pull request by cchampet**
* Up avtranscoder to v0.5.1
* Set log level of avreader and avwriter to QUIET
* Move avtranscoder submodule with AudioVideo plugin
* Deploy ffmpeg libraries if avTranscoder is built

### CODE REVIEW

**Comment by cchampet** [comment]
Ping @fabiencastan
Can you test ButtleOFX project with this PR (see last commit, about problem with log of libav ;) ) ?

File: plugins/image/io/AudioVideo/src/writer/AVWriterPluginFactory.cpp [hunk 1]
```diff
@@ -438,7 +439,7 @@ void AVWriterPluginFactory::describeInContext( OFX::ImageEffectDescriptor& desc,
 		audioOffsetName << "_" << indexAudioInput;
 		OFX::IntParamDescriptor* audioOffsetParam = desc.defineIntParam( audioOffsetName.str() );
 		audioOffsetParam->setLabel( "Offset" );
-		audioOffsetParam->setHint( "Add an offset (in milliseconds) at the beginning of the stream. By default 0." );
+		audioOffsetParam->setHint( "Add an offset (in seconds) at the beginning of the stream. By default 0." );
```
Comment (fabiencastan) [inline plugins/image/io/AudioVideo/src/writer/AVWriterPluginFactory.cpp]: If it's in seconds (instead of milliseconds) it should not be an Integer param.

Comment (cchampet) [inline plugins/image/io/AudioVideo/src/writer/AVWriterPluginFactory.cpp]: Corrected 6ed27e7

**Comment by cchampet** [comment]
Can be merged ? @fabiencastan

File: plugins/image/io/AudioVideo/src/writer/AVWriterPluginFactory.cpp [hunk 2]
```diff
@@ -433,14 +434,14 @@ void AVWriterPluginFactory::describeInContext( OFX::ImageEffectDescriptor& desc,
...
-		// add audio channel index
+		// add audio offset
 		std::ostringstream audioOffsetName( kParamAudioOffset, std::ios_base::in | std::ios_base::ate );
 		audioOffsetName << "_" << indexAudioInput;
-		OFX::IntParamDescriptor* audioOffsetParam = desc.defineIntParam( audioOffsetName.str() );
+		OFX::DoubleParamDescriptor* audioOffsetParam = desc.defineDoubleParam( audioOffsetName.str() );
 		audioOffsetParam->setLabel( "Offset" );
-		audioOffsetParam->setHint( "Add an offset (in milliseconds) at the beginning of the stream. By default 0." );
+		audioOffsetParam->setHint( "Add an offset (in seconds) at the beginning of the stream. By default 0." );
 		audioOffsetParam->setRange( 0, INT_MAX );
```
Comment (fabiencastan) [inline plugins/image/io/AudioVideo/src/writer/AVWriterPluginFactory.cpp]: Audio offset could be positive of negative, isn't it?

Comment (cchampet) [inline plugins/image/io/AudioVideo/src/writer/AVWriterPluginFactory.cpp]: #373

**Pull request merged**